### DISCUSSION TITLE
Run multi threaded tests

### DISCUSSION THREAD

**Discussion by deepakgauda-dk** [artilleryio/artillery#956] [Artillery]
Hello friends,

I would like to run a load test with multiple thread scenario. Please let me know how this can be achieved using artillery.
I am using artillery version 1.6.2 and node.js version v10.16.0 in windows.

P.S: is defining "arrivalRate" more than 1 would work for this objective ?

**Comment by hassy** [comment]
hi @deepakgauda-dk 👋  If you mean "threads" in the JMeter sense, then yes, an arrivalRate of 2 for example will launch two virtual users in parallel.
